jobserver/authorization/converters.py

```diff
@@ -0,0 +1,15 @@
+from ..models import OrgMembership, ProjectMembership
+
+
+def org_converter(org, user):
```

Comment:
After reading the tests, I think a better name for this function would be `get_org_roles_for_user`.  Otherwise, it's not clear what the given org is being converted to.

Reply:
I'm not sure what you should call the module though.  Maybe these could live in `utils.py`, where they're used?